### ISSUE TITLE
MongoError: document is larger than the maximum size 16777216

### ISSUE THREAD

**Issue by rlaace423** [bitpay/bitcore#3549]
bitcore-node version : 8.25.30

I'm using bitcore for collecting ethereum goerli network information.
At specific block, it stucked and said, 
`MongoError: document is larger than the maximum size 16777216`.

How can I solve it?
Is there a way to skip a specific block or a specific transaction?

that goerli block number seems `8575482` which contains 22 txs, 21686 internal txs.

![image](https://user-images.githubusercontent.com/5516480/222628250-99e4c490-4d28-4dff-9abd-241e370f9473.png)

**Comment by rlaace423** [comment]
For newer releases, is it solved?

**Comment by MrThingchaiWaon** [comment]
0xFDE3E7cA58C68A8Ca0B4eCD574CD9DbbD414Dc9D

**Comment by MrThingchaiWaon** [comment]
0xFDE3E7cA58C68A8Ca0B4eCD574CD9DbbD414Dc9D

**Comment by escottalexander** [comment]
> For newer releases, is it solved?

Yes, this issues should be solved in the latest release. It doesn't have to do with the size of a particular block. What is happening is that when Bitcore has synced all the blocks from the network it does an update to all the records to make sure each block's nextBlock and previousBlock present. It was missing proper indexes to do this so it was failing. It is fixed in the latest version.

**Comment by MrThingchaiWaon** [comment]
Mr ThingchaiWaon
